site/content/en/docs/enterprise/segment-anything-2-tracker.md

```diff
@@ -8,24 +8,54 @@ description: 'Accelerating video labeling using SAM2 model'
 ## Overview
 
 Segment Anything 2 is a segmentation model that allows fast and precise selection of any object in videos or images.
-For enterprise customers, this model can be installed in their self-hosted solution. To ensure a good experience,
-it is strongly recommended to deploy the model using a GPU. Although it is possible to use a CPU-based version,
-it generally performs much slower and is suitable only for handling a single parallel request. Unlike a regular
-tracking model, the SAM 2 tracker is implemented as an annotation action. This allows it to be applied to existing
-objects (polygons and masks) to track them forward for a specified number of frames.
+SAM2 tracking is available in two implementations:
+
+1. **Classic SAM2 Tracker (Annotation Action)**: Available for Enterprise self-hosted Basic/Premium deployments.
+This is implemented as an annotation action using serverless functions and requires Nuclio deployment.
+
+1. **AI Agent SAM2 Tracker**: Available for CVAT Online and Enterprise 2.42.0+ via auto-annotation (AA) functions
+that run on user-side agents. This brings SAM2 tracking capabilities to CVAT Online users who previously
+couldn't access this feature.
+
+For self-hosted enterprise customers using the classic implementation, it is strongly recommended
+to deploy the model using a GPU. Although it is possible to use a CPU-based version,
+it generally performs much slower and is suitable only for handling a single parallel request.
+The AI agent variant runs on user hardware, providing flexibility for GPU usage without
+server configuration requirements.
+
+Unlike a regular tracking model, both SAM2 tracker implementations are designed to be applied
+to existing objects (polygons and masks) to track them forward for a specified number of frames.
+
+## Platform Availability and Deployment Options
+
+The following table helps you choose the appropriate SAM2 tracking implementation based on your deployment:
+
+| Feature | Classic SAM2 Tracker | AI Agent SAM2 Tracker |
+| ------- | -------------------- | --------------------- |
+| **Platform** | Enterprise self-hosted Basic/Premium | CVAT Online + Enterprise 2.42.0+ |
```

Comment:
In the patch you mean Enterprise by different terms:

- Enterprise self-hosted
- Enterprise self-hosted Basic/Premium
- Just Enterprise

Are there any reasons to write "self-hosted" or "Basic/Premium" when we are talking about Enterprise?